extensions/dicom-pdf/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@ohif/extension-dicom-pdf",
+  "version": "0.0.1",
```

Comment:
@swederik Talking with Alireza, we discussed just changing all of the extensions to 3.x.x versions.  Will have a separate PR for that.